projects/packages/my-jetpack/_inc/components/product-cards-section/boost-card/index.tsx

```diff
@@ -1,11 +1,10 @@
 import { __ } from '@wordpress/i18n';
-import PropTypes from 'prop-types';
 import React from 'react';
 import ProductCard from '../../connected-product-card';
 import { PRODUCT_STATUSES } from '../../product-card/action-button';
 import BoostSpeedScore from './boost-speed-score';
 
-const BoostCard = ( { admin }: { admin: boolean } ) => {
+const BoostCard: React.FC< { admin: boolean } > = ( { admin } ) => {
```

Comment:
Nitpick, but I prefer importing the FC component like this

```tsx
import type { FC } from 'react'
```

just because it looks a bit cleaner. That is opinionated of me though, so I'm just leaving this as a personal note 😅